Server_db/controllers/controller.js

```diff
@@ -134,6 +134,59 @@ class Controller {
 
     return util.successStatus(res, 200, 'message', 'Account successfully deleted');
   }
+
+  /**
+    * @static
+    * @description Allows Staff to debit an account
```

Comment:
Your description is not explanatory enough
Because i can see you are implementing both debit and credit in this feature